pkg/model/spotinstmodel/instance_group.go

```diff
@@ -34,6 +33,10 @@ import (
 )
 
 const (
+	// InstanceGroupLabelManaged is the metadata label used on the instance
+	// group to specify that the Spotinst provider should be used to upon creation.
+	InstanceGroupLabelManaged = "spotinst.io/managed"
```

Comment:
This is a cool idea.  One thing we could also do (with the move to adopt cluster-api), is to actually create a "provider" or "manager" or "strategy" field on the InstanceGroup.  My personal choice would be `provider`, but I don't have strong feelings.

Reply:
Yeah, I agree. It'd be great.

Reply:
BTW, do you think we should change the metadata label to `spotinst.io/hybrid: "true"` (instead of `spotinst.io/managed: "true"`) so it will better match the feature flag?

Reply:
I do like the idea of matching the feature flag there

Reply:
Well, we already have customers that use this feature using a custom build of kops and we prefer to not introduce a breaking change by renaming the label. So, we've changed the implementation to support both cases.